Storage/RemoteDevice.swift

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+import Account
+
+public struct RemoteDevice {
```

Comment:
Make `RemoteDevice` the base struct of `FxADevice`, put `RemoteDevice` in `Shared`, probably in a new file, change the protocol to take `RemoteDevice` instead of `FxADevice`, and then you can kill the `import Account` throughout `Storage`, which will keep our incremental build times down.